src/main/kotlin/no/nav/api/dialog/saf/SafClient.kt

```diff
@@ -1,42 +1,45 @@
 package no.nav.api.dialog.saf
 
+import com.expediagroup.graphql.client.types.GraphQLClientResponse
+import io.ktor.client.*
 import io.ktor.client.engine.*
 import io.ktor.client.engine.okhttp.*
 import io.ktor.client.request.*
-import no.nav.api.dialog.saf.queries.HentBrukerssaker
+import no.nav.api.generated.saf.HentBrukerssaker
+import no.nav.api.generated.saf.enums.BrukerIdType
+import no.nav.api.generated.saf.inputs.BrukerIdInput
 import no.nav.utils.*
+import java.net.URL
 
 class SafClient(
     private val safUrl: String,
     private val tokenclient: BoundedMachineToMachineTokenClient,
     httpEngine: HttpClientEngine = OkHttp.create()
 ) {
-    private val graphqlClient = GraphQLClient(
-        httpClient = GraphQLClient.createHttpClient(httpEngine),
-        config = GraphQLClientConfig(
-            serviceName = "SAF",
-            critical = false,
-            requestConfig = {
-                val token = tokenclient.createMachineToMachineToken()
-                url(safUrl)
-                header("Authorization", "Bearer $token")
-                header("X-Correlation-ID", getCallId())
-                header("Content-Type", "application/json")
-            }
-        )
+    private val graphqlKtorClient = LoggingGraphQLKtorClient(
```

Comment:
"ktor" i variabelnavnet gir oss egentlig ikke noe nyttig info, så tenker vi egentlig kan holde navnene litt enklere. 
```suggestion
    private val graphqlClient = LoggingGraphQLKtorClient(
```